pkg/gatherers/clusterconfig/machine_configs.go

```diff
@@ -13,17 +13,30 @@ import (
 	"github.com/openshift/insights-operator/pkg/record"
 )
 
-// GatherMachineConfigs collects MachineConfigs definitions. Following data is intentionally removed from the definitions:
-// * `spec.config.storage.files`
-// * `spec.config.passwd.users`
+// GatherMachineConfigs Collects MachineConfigs definitions. Following data is intentionally removed from the definitions:
+// - `spec.config.storage.files`
+// - `spec.config.passwd.users`
 //
-// Response see https://docs.openshift.com/container-platform/4.7/rest_api/machine_apis/machineconfig-machineconfiguration-openshift-io-v1.html
+// ### API Reference
+// - https://docs.openshift.com/container-platform/4.7/rest_api/machine_apis/machineconfig-machineconfiguration-openshift-io-v1.html
 //
-// * Location in archive: config/machineconfigs/{name}.json
-// * Id in config: clusterconfig/machine_configs
-// * Since versions:
-//   - 4.8.5+
-//   - 4.9+
+// ### Sample data
+// - docs/insights-archive-sample/config/machineconfigs/75-worker-sap-data-intelligence.json
+//
+// ### Location in archive
+// - `config/machineconfigs/{resource}.json`
+//
+// ### Config ID
+// `clusterconfig/machine_configs`
+//
+// ### Released version
+// - 4.8.2
+//
+// ### Backported versions
+// None
```

Comment:
It's missing `4.8.5`

Reply:
Do you have the PR, BZ, or RN that confirms it?

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1977754

Reply:
It seems the release version was 4.9.0 in fact https://errata.devel.redhat.com/advisory/81213

Reply:
This is a very good piece of information. Thank you. I'll adjust.